src/algoliasearch.helper.js

```diff
@@ -144,7 +144,12 @@ inherits(AlgoliaSearchHelper, events.EventEmitter);
  * @chainable
  */
 AlgoliaSearchHelper.prototype.search = function() {
-  this._search();
+  this._search({onlyWithDerivedHelpers: false});
+  return this;
+};
+
+AlgoliaSearchHelper.prototype.searchOnlyWithDerivedHelpers = function() {
```

Comment:
I was thinking about not exposing this one externally, and relying on `._search({onlyWithDerivedHelpers: false})` in InstantSearch, but that probably doesn't really bring much value

Reply:
We can avoid exposing a function but we can't use your solution. For the federated search implementation, we have to override the `.search` method of the children Helper to bubbles up until the root. With the private call, we can't easily intercept the call anymore. What we could do is to expose an option rather than a dedicated method:

```js
AlgoliaSearchHelper.prototype.search = function(options) {
  this._search(options);
  return this;
};
```

I don't think, we'll document the method anyway. This solution is short term. Once we have the federated search implementation, it would be a good opportunity to redesign a "Helper" with a first-class support of it. At the end both implementations are equivalent. The first one is a bit more explicit IMO.